web-react-ts/src/permission-utils.ts

```diff
@@ -173,8 +177,12 @@ export const permitArrivalsCounter = (): Role[] => {
 export const permitArrivalsConfirmer = (): Role[] => {
   return ['arrivalsConfirmerStream']
 }
-export const permitArrivalsHelpers = (): Role[] => {
-  return ['arrivalsCounterStream', 'arrivalsConfirmerStream']
+export const permitArrivalsHelpers = (churchLevel: ChurchLevel): Role[] => {
```

Comment:
There is only one churchLevel we are expecting here, and that is stream. Please adjust accordingly